### PR TITLE
Upgrade to polkadot v1.1.0

### PR DESCRIPTION
**Pull request by rimbi**
Upgrades dependencies to the v1.1.0 of new polkadot-sdk.

### CODE REVIEW

**Comment by xlc** [comment]
polkadot-sdk is now published to crates.io so we may want to switch to those and then we will be able to publish ORML to crates.io as well.
But there are still some unanswered questions about polkadot-sdk's publish process so we could do this meanwhile.

File: asset-registry/Cargo.toml [hunk 1]
```diff
@@ -8,43 +8,45 @@ authors = ["Interlay Ltd, etc"]
...
 # substrate
-frame-support = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
-frame-system = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
-sp-std = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
+frame-support = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+frame-system = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+sp-io = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+sp-runtime = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+ sp-std = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
```
Comment (xlc) [inline asset-registry/Cargo.toml:21]:
```suggestion
sp-std = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
```

File: benchmarking/Cargo.toml [hunk 2]
```diff
@@ -9,24 +9,25 @@ edition = "2021"
...
+frame-support = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+sp-api = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+sp-io = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+sp-runtime = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+sp-runtime-interface = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+sp-std = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
+sp-storage = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false , branch = "release-polkadot-v1.1.0" }
 
 [dev-dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false }
```
Comment (xlc) [inline benchmarking/Cargo.toml:27]: why this in dev-dependencies? codec is already in the dependencies

File: xtokens/src/mock/mod.rs [hunk 3]
```diff
@@ -338,11 +339,60 @@ impl WeightTrader for AllTokensAreCreatedEqualToWeight {
...
 }
+
+/// Allows execution from all origins taking payment into account.
+///
+/// Only allows for `TeleportAsset`, `WithdrawAsset`, `ClaimAsset` and
+/// `ReserveAssetDeposit` XCMs because they are the only ones that place assets
+/// in the Holding Register to pay for execution. This is almost equal to
+/// [`xcm_builder::AllowTopLevelPaidExecutionFrom<T>`] except that it allows for
+/// multiple assets and is not generic to allow all origins.
+pub struct AllowTopLevelPaidExecution;
```
Comment (xlc) [inline xtokens/src/mock/mod.rs:358]: Add comment explain why this is duplicated and link to upstream issue https://github.com/paritytech/polkadot-sdk/issues/1638

**Review by xlc — CHANGES_REQUESTED** [review]

File: benchmarking/Cargo.toml (hunk 2 above)
Comment (rimbi) [reply benchmarking/Cargo.toml]: done 👍

File: xtokens/src/mock/mod.rs (hunk 3 above)
Comment (rimbi) [reply xtokens/src/mock/mod.rs:360]: done 👍

**Review by xlc — APPROVED** [review]

**Pull request merged**